internal/server/execclient/client.go

```diff
@@ -142,7 +142,7 @@ func (c *Client) Run() (int, error) {
 
 	if ptyF != nil {
 		// We need to go into raw mode with stdin
-		if f, ok := c.Stdin.(*os.File); ok {
+		if f, ok := c.Stdout.(*os.File); ok {
```

Comment:
This makes sense but also seems suspect. We have been using this code for some time with ptys with no problem.(even executing stuff like `vim` and resizing windows). Why this change?

Reply:
The commit message on that line explains the bug. Basically it has to do with someone doing `date | waypoint exec`.

Reply:
Hm hm yes. Okay, just verify `vim` and friends still work.